src/xcode/ENA/ENA/Source/Workers/Logging/Logging.swift

```diff
@@ -122,9 +122,9 @@ enum Log {
 			// swiftlint:disable:next no_direct_oslog
 			os_log("%{public}@ %{public}@", log: log, type: type, meta, message)
 		}
-
+		
 		// Save logs to File. This is used for viewing and exporting logs from debug menu.
-		fileLogger.log(message, logType: type, file: file, line: line, function: function)
+		fileLogger.log(message, logType: type, file: file, line: line, function: function, error: error?.localizedDescription)
```

Comment:
For os_log we log the error itself as well as the localized description, as we sometimes share the localized description between multiple errors and often don't have all the error's details included